src/gei/Commands/GenerateScriptCommand.cs

```diff
@@ -435,6 +448,18 @@ exit 1
             return content.ToString();
         }
 
+        private async Task<IEnumerable<string>> GetTeamProjectRepos(AdoApi adoApi, string adoOrg, string teamProject)
+        {
+            _log.LogInformation($"Team Project: {teamProject}");
```

Comment:
duplicate log lines with line 232

Reply:
removed from earlier line so that both branches are covered by the method logging